### DISCUSSION TITLE
Question: How to set a value for pageRedirectUri in info.ftl

### DISCUSSION THREAD

**Discussion by mallowigi** [keycloak/keycloak#19603] [Q&A]
I've been trying to scourge all parts of the internet to find a way to set the `pageRedirectUri` variable, which is found in `info.ftl`. I even tried with ChatGPT but to no success.

I am using keycloak version 21 and I want to be able to customize the redirect url once the user clicks to the "forgot password" link in his email. Currently once he finishes updating the password, he is redirected to the login page of keycloak, but for the `account` client_id, and I don't want that. Instead I want him to be registered to my application home page/login page.

Thanks in advance for any help...

**Comment by joeykmh** [comment]
It seems you can configure this in the `account` client of your realm by adjusting the **Home URL**:

<img width="1702" alt="Screenshot 2024-03-15 at 14 30 42" src="https://github.com/keycloak/keycloak/assets/7550957/50f16542-cf4b-4af5-b095-0ec4dea549d4">

This might have unintended consequences elsewhere 🤷 , but I could not find a better way to set the redirect URI short of overriding the `info.ftl` template entirely.

Edit:
In my case I was using the Java SDK directly and calling `UserResource.sendVerifyEmail()` to send an email verification manually. In this method you can pass an optional `client` id (e.g. `UserResource.sendVerifyEmail("my-client")` which will then use _that_ client's **Home URL** instead of the built-in `account` client. This is preferable for me, because I don't know what other unintended consequences changing the **Home URL of `account` will have.
